Formula/a/agda.rb

```diff
@@ -2,14 +2,16 @@ class Agda < Formula
   desc "Dependently typed functional programming language"
   homepage "https://wiki.portal.chalmers.se/agda/"
   license "BSD-3-Clause"
+  revision 1
 
   stable do
     url "https://hackage.haskell.org/package/Agda-2.6.4.1/Agda-2.6.4.1.tar.gz"
     sha256 "23248a9b3c50c81ea4751518a66f2a6144617b6a5a9202686b54e33bc9cbd080"
 
     resource "stdlib" do
-      url "https://github.com/agda/agda-stdlib/archive/refs/tags/v1.7.3.tar.gz"
-      sha256 "91c42323fdc94d032a8c98ea9249d9d77e7ba3b51749fe85f18536dbbe603437"
+      url "https://github.com/agda/agda-stdlib.git",
+          tag:      "v2.0",
+          revision: "2b8fff10f4033b91a6df4007e4a65cb10047c89c"
```

Comment:
Can we use the tarball instead of the git tag?

Reply:
We totally can, but I wasn't sure if audit liked that anymore; it's told me off for it before.

Reply:
Should be fine to use the tarball then. I'll take a look if there is some kind of audit failure. Thanks!